pkg/scheduler/core/extender.go

```diff
@@ -114,6 +115,36 @@ func NewHTTPExtender(config *schedulerapi.ExtenderConfig) (algorithm.SchedulerEx
 	}, nil
 }
 
+// Equal is used to check if two extenders are equal
+// ignoring the client field, exported for testing
+func Equal(extender1, extender2 *HTTPExtender) bool {
+	if extender1.extenderURL != extender2.extenderURL {
+		return false
+	}
+	if extender1.preemptVerb != extender2.preemptVerb {
+		return false
+	}
+	if extender1.prioritizeVerb != extender2.prioritizeVerb {
+		return false
+	}
+	if extender1.bindVerb != extender2.bindVerb {
+		return false
+	}
+	if extender1.weight != extender2.weight {
+		return false
+	}
+	if extender1.nodeCacheCapable != extender2.nodeCacheCapable {
+		return false
+	}
+	if !reflect.DeepEqual(extender1.managedResources, extender2.managedResources) {
+		return false
+	}
+	if extender1.weight != extender2.weight {
+		return false
+	}
+	return true
```

Comment:
Did you miss `ignorable`? prefer to use the same order for the comparisons as the struct definition.

Reply:
They have the same order, just missed this one (a copy paste error :man_facepalming:)